mgmt/src/grpc/converter/old_impl.rs

```diff
@@ -816,6 +812,10 @@ pub fn convert_bgp_neighbor_to_grpc(
     })
 }
 
+fn has_redistribute(redistribute: &[Redistribute], protocol: &Protocol) -> bool {
+    redistribute.iter().any(|r| r.protocol == *protocol)
+}
+
```

Comment:
Nit: This is added in the first commit, but should it be added later? It's not used yet in the first commit.

Reply:
Yep this should be moved, will do.

Reply:
Fixed.